03-bash/task1.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+is_integer () {
+	local re='^[0-9]+$'
+	[[ $1 =~ $re ]]
+	return $?
```

Comment:
return $? is not required. $? is returned by default

Reply:
As I understand $? is a result of [[ $1 =~ $re ]] command. So, I prefer to return explicit value